This repository includes the custom packages and scripts used in "Leveraging phylogenetic signal to unravel microbial community function and assembly rules" by Talavera-Marcos, Parras-Moltó and Aguirre de Cárcer (https://doi.org/10.21203/rs.3.rs-2272005/v1)

The approaches used for each section of the paper are listed down below

## Folders and repositories for each section

- PCGs in the environment: [BacterialCore.py](https://github.com/mparmol/BacterialCore)
- Experimental dataset processing: [16S](https://github.com/silvtal/16S/tree/main/goldford)
- Mapping 16S sequences to genomes and metabolic modeling: [TFM](https://github.com/silvtal/TFM)
- Core pangenome exploration: [get_core_reactions](https://github.com/silvtal/get_core_reactions)
- Drift modeling:
  - [R package for dilution-growth simulations](https://github.com/silvtal/dilgrowth)
  - [Specific datasets and analysis](https://github.com/silvtal/phyloassembly/blob/main/simulations)
- Interactions between populations: [Correlation graphs](https://github.com/silvtal/correlation_graphs)


## Additional repositories
- outliers_search.R (generates Suppl. Fig. 4)
- Plotting FBA fluxes (Fig. 3 and Suppl. Fig. 2): [SBML_to_graph](https://github.com/silvtal/SBML_to_graph)
